fblldbobjcruntimehelpers.py

```diff
@@ -32,6 +32,10 @@ def class_getSuperclass(klass):
   value = fb.evaluateExpression(command)
   return value
 
+def class_isMetaClass(klass):
+    command = '(BOOL)class_isMetaClass((Class){})'.format(klass)
+    return fb.evaluateBooleanExpression(command)
```

Comment:
@longv2go You don't need the `(BOOL)` cast when calling `evaluateBooleanExpression`.